tests/kuttl/test-clowdappref/03-config-assert.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    # Get the consumer-app config and verify it contains the ClowdAppRef endpoints
+    kubectl get secret consumer-app -n test-clowdappref -o jsonpath='{.data.cdappconfig\.json}' | base64 -d > /tmp/consumer-app-config.json
+    echo "Consumer app config:"
+    cat /tmp/consumer-app-config.json
+    
+    # Check that remote-service endpoints are present
+    grep -q "auth.remote-cluster.example.com" /tmp/consumer-app-config.json
```

Comment:
The greps are good to check the presence of these strings ... but I think it would be better to assert that the JSON structure for this app in the cdappconfig.json looks like you would expect it to. For that, I'd probably use some jq asserts similar to https://github.com/RedHatInsights/clowder/blob/master/tests/kuttl/test-basic-app/02-json-asserts.yaml#L5-L23